CMakeLists.txt

```diff
@@ -174,6 +174,12 @@ if(CMAKE_C_COMPILER_ID MATCHES "^Intel")
         set(WARNFLAGS_MAINTAINER /W5)
         set(WARNFLAGS_DISABLE)
     endif()
+    check_c_compiler_flag(-diag-disable=10441 HAVE_DIAG_10441)
+    if(HAVE_DIAG_10441)
+        list(APPEND WARNFLAGS_DISABLE "-diag-disable=10441")
+        list(APPEND CMAKE_EXE_LINKER_FLAGS "-diag-disable=10441")
+        list(APPEND CMAKE_SHARED_LINKER_FLAGS "-diag-disable=10441")
```

Comment:
``CMAKE_EXE_LINKER_FLAGS`` and ``CMAKE_SHARED_LINKER_FLAGS`` is not CMake list (comma separated). It is space separated string with cli arguments for linker or compiler.

Reply:
Changed.